gen.py

```diff
@@ -18,33 +18,35 @@
 from subprocess import PIPE, Popen, call
 try:
     from gi import require_version
-    require_version('Rsvg', '2.0')
+    require_version("Rsvg", "2.0")
     from gi.repository import Rsvg
     from cairo import ImageSurface, Context, FORMAT_ARGB32
     use_inkscape = False
 except (ImportError, AttributeError, ValueError):
-    ink_flag = call(['which', 'inkscape'], stdout=PIPE, stderr=PIPE)
-    if ink_flag == 0:
-        use_inkscape = True
-    else:
+    # Assigns true if Inkscape found, false otherwise
+    use_inkscape = not call(["which", "inkscape"], stdout=PIPE, stderr=PIPE)
+    if not use_inkscape:
```

Comment:
Rather than using distinct `flag` and `use` variables, can just use one with logical `not` converters.